pkgs/applications/networking/browsers/palemoon/default.nix

```diff
@@ -137,24 +140,15 @@ stdenv.mkDerivation rec {
 
     ./mach install
 
-    # Fix missing icon due to wrong WMClass
-    # https://forum.palemoon.org/viewtopic.php?f=3&t=26746&p=214221#p214221
-    substituteInPlace ./palemoon/branding/official/palemoon.desktop \
-      --replace 'StartupWMClass="pale moon"' 'StartupWMClass=Pale moon'
+    # Install official branding stuff (desktop file & icons)
     desktop-file-install --dir=$out/share/applications \
-      ./palemoon/branding/official/palemoon.desktop
-
-    # Install official branding icons
+      ./other-licenses/branding/palemoon/official/palemoon.desktop
     for iconname in default{16,22,24,32,48,256} mozicon128; do
       n=''${iconname//[^0-9]/}
       size=$n"x"$n
-      install -Dm644 ./palemoon/branding/official/$iconname.png $out/share/icons/hicolor/$size/apps/palemoon.png
+      install -Dm644 ./other-licenses/branding/palemoon/official/$iconname.png $out/share/icons/hicolor/$size/apps/palemoon.png
```

Comment:
What does `other-licenses` mean here?
Yes, it is just a directory name, but is there something we should pay attention here?

Reply:
All branding ("official", "beta", "unstable" etc) except the MPL2.0-licensed "unofficial" one (which is included for building convenience and intended to be replaced for distribution) has been moved to `other-licenses/branding/palemoon` in this major release.

The licensing situation itself is unchanged: The branding is (and always have been) under a [non-MPL2.0 license](https://repo.palemoon.org/MoonchildProductions/Pale-Moon/src/branch/master/other-licenses/branding/palemoon/LICENSE) that gives Moonchild Productions exclusive ownership of the branding & "Pale Moon" name. [This post](https://forum.palemoon.org/viewtopic.php?f=24&t=26031) outlines when the official branding may be used for distributed source builds.

tl;dr just the same as always w/r/t permitted compilers, mozconfig settings, use of in-tree libraries, no modifications to defaults etc.

Reply:
OK then!